modules/apps/remote-web-app/remote-web-app-web/src/main/java/com/liferay/remote/web/app/web/internal/portlet/RemoteWebAppAdminPortlet.java

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.remote.web.app.web.internal.portlet;
+
+import com.liferay.portal.kernel.portlet.bridges.mvc.MVCPortlet;
+import com.liferay.remote.web.app.web.internal.constants.RemoteWebAppPortletKeys;
+
+import javax.portlet.Portlet;
+
+import org.osgi.service.component.annotations.Component;
+
+/**
+ * @author Iván Zaera
+ */
+@Component(
+	immediate = true,
+	property = {
+		"com.liferay.portlet.css-class-wrapper=portlet-remote-web-app-admin",
+		"com.liferay.portlet.display-category=hidden",
+		"com.liferay.portlet.header-portlet-css=/remote_web_app_admin/css/main.css",
+//		"com.liferay.portlet.icon=/wiki/icons/wiki.png",
```

Comment:
This was obviously copied from `wiki`. I left some comments to remember us that we may add more functionality.